src/libraries/Microsoft.Extensions.Logging.EventLog/src/EventLogLogger.cs

```diff
@@ -164,6 +164,8 @@ private void WriteMessage(string message, EventLogEntryType eventLogEntryType, i
             }
         }
 
+#pragma warning disable CA1416 // Debug.Assert(OperatingSystem.IsWindows()) is not available
```

Comment:
@buyaa-n is there a bug we can reference in this comment?

Reply:
This is because the assembly targets netstandard, so the relevant APIs aren't available

Reply:
Ah, thanks! I _think_ `Debug.Assert(RuntimeInformation.IsOSPlatform(OSPlatform.Windows))` will work in this case then.

Reply:
Yes that should work